File: 2-semestre/uri-online-judge/c/1114.c
#include <stdio.h>
 
int main() {
    int senha;
    
    do {
        scanf("%d\n", &senha);
        
        if (senha != 2002) printf("Senha Invalida\n");
        if (senha == 2002) printf("Acesso Permitido\n");
    } while (senha != 2002);
 
    return 0;
}

File: 2-semestre/uri-online-judge/java/1114.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
 
    public static void main(String[] args) throws IOException {
        int senha;
        
        Scanner entrada = new Scanner(System.in);
        
        do {
            senha = entrada.nextInt();

            System.out.printf(senha != 2002 ? "Senha Invalida\n" : "Acesso Permitido\n");
        } while (senha != 2002);
    }
}
